src/python/pants/task/task.py

```diff
@@ -198,9 +211,13 @@ def get_passthru_args(self):
     :API: public
     """
     if not self.supports_passthru_args():
-      raise TaskError('{0} Does not support passthru args.'.format(self.stable_name()))
-    else:
-      return self.context.options.passthru_args_for_scope(self.options_scope)
+      raise TaskError('{} Does not support passthru args.'.format(self.stable_name()))
+
+    passthru_args = []
+    for arg in self.get_options().get('passthrough_args', default=()):
+      passthru_args.extend(safe_shlex_split(arg))
```

Comment:
Given that this is arriving as a list already, is it necessary to support splitting them this way? Or is this a harmless convenience?

Reply:
It seems like a not-so-harmless convenience. Either way there are compsomises. I used the pytest casehere for example.

With the old shlexing:
```
./pants test.pytest --options="-k 'test_reset or test_singleton'" tests/python/pants_test/base:build_root
```

Without:
```
./pants test.pytest --passthrough-args="-k test_reset or test_singleton" tests/python/pants_test/base:build_root
```

Without is a bit odd, but using `--passthrough-args` as a flag instead of as an env var or ini option is also odd - you can just use `--`. Presumably the without form breaks Twitter configs, but since this is a new option requiring a migration anyway, I'm happy to simplify. No functionality is lost and using `passthrough_args` in an ini file is natural as a result.